src/libraries/System.Private.CoreLib/src/System/IndexOfAnyAsciiSearcher.cs

```diff
@@ -0,0 +1,353 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+
+namespace System
+{
+    internal static class IndexOfAnyAsciiSearcher
+    {
+        private static bool IsSupported => Ssse3.IsSupported || AdvSimd.Arm64.IsSupported;
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static unsafe bool TryComputeBitmap(ReadOnlySpan<char> values, byte* bitmap, out bool needleContainsZero)
+        {
+            byte* bitmapLocal = bitmap;
```

Comment:
Why is the local copy needed?  I seem to remember there being an issue about code gen? If so, can we include a link to the open issue?

Reply:
I copied this approach from
https://github.com/dotnet/runtime/blob/fbc20578253e0deb7267eea5a02b680c3cfe0e2a/src/libraries/System.Private.CoreLib/src/System/ProbabilisticMap.cs#L40

I'll add a comment here as well